packages/dd-trace/src/plugins/util/web.js

```diff
@@ -221,6 +221,20 @@ const web = {
       const context = contexts.get(req)
       context.error = context.error || error
     }
+  },
+
+  setUser (req, user) {
```

Comment:
The user id is mandatory in the spec so I rather suggest adding as an explicit mandatory argument.